src/memory/AmdFlash.hh

```diff
@@ -316,8 +337,29 @@ private:
 		BF8<1> sectorLock;
 		BF8<0> continuity;
 	} statusRegister;
+
+	struct EraseOperation final {
+		size_t index = 0;
+		std::vector<uint8_t> buffer;
```

Comment:
Is this conceptually meant as a `vector<bool>`? I know that `vector<bool>` is weird in c++, and that you might want to avoid it for that reason. But maybe still document that as such?
```
std::vector</*bool*/uint8_t> buffer;
```

Reply:
The serialisation doesn’t like vector<bool>.